### ISSUE TITLE
Improve footer design

### ISSUE THREAD

**Issue by pmackay** [folklabs/openlocal#64]
@djwesto do we want to pin it to the bottom similar to Chimni?

**Comment by djwesto** [comment]
Yes, absolutely. But do you recall the problems we had with page content elements overlapping the pinned footer on Chimni? We want to avoid that this time.

**Comment by djwesto** [comment]
For some reason I can't edit the Description field on this issue so I'll add the task list here:
![image](https://cloud.githubusercontent.com/assets/702685/22468079/ffba8b30-e7be-11e6-9c35-a0c28e4ffd6c.png)

- [ ] Replace User log in fields from footer (it'll cause HTTPS issues) with simple Login link
- [ ] Remove Who's new list and Who's online
- [ ] Run main navigation menu horizontally instead of vertically
- [ ] Add second tier with copyright info and other basic info

**Comment by pmackay** [comment]
Partly I was considering theming. Those points above are valid but may come later at site launch time, rather than features baked into OL.
